packages/react/src/LabelGroup/LabelGroup.tsx

```diff
@@ -73,15 +73,14 @@ const InlineToggle: React.FC<{
   hiddenItemIds,
   isOverflowShown,
   showAllTokensInline,
-  totalLength,
 }) =>
   isOverflowShown ? (
     <Button ref={collapseButtonRef} onClick={collapseInlineExpandedChildren} size="small" variant="invisible">
       Show less
     </Button>
   ) : hiddenItemIds.length ? (
     <Button ref={expandButtonRef} variant="invisible" size="small" onClick={showAllTokensInline}>
-      <VisuallyHidden>Show all {totalLength}</VisuallyHidden>
+      <VisuallyHidden>Show +{hiddenItemIds.length} more</VisuallyHidden>
```

Comment:
Random question, what does including `+` here do? Is this changing how it's announced?

Reply:
It doesn't really change much. For all the screen readers I tested with (NVDA, VoiceOver, JAWS) they all announced "Show plus 11 more". It would be preferable for it to say "11 plus", but then we'd need to move the + in the visual text to be after the number (e.g. 11+).